agenta-backend/agenta_backend/models/db_models.py

```diff
@@ -6,6 +6,7 @@
 class OrganizationDB(Model):
     name: str = Field(default="agenta")
     description: str = Field(default="")
+    members: Dict["UserDB", str]
```

Comment:
Why use a Dict here, why not list[UserDB]? @aybruhm what do you think?

Reply:
@mmabrouk from the PR comment, I did this so that we can assign roles to users.

Reply:
This is the right way to do it.

```suggestion
members: Dict[str, UserDB]
```

Reply:
changed it to `List[Dict["UserDB", str]]`

Reply:
The right way to do this is to not have an admin or owner together with the members. It should be:

```suggestion
owner: UserDB
members: Optional[List[UserDB]]
```

cc: @mmabrouk

Reply:
Yes, I thought about this but considered that when we have to query members, it means we'll query members then get the admin separately or show them separately but in the same page.

Reply:
I don't think that's a problem. It's a simple query

Reply:
> Yes, I thought about this but considered that when we have to query members, it means we'll query members then get the admin separately or show them separately but in the same page.
> 
> @aybruhm @mmabrouk

You won't be. Once you've gotten the organization instance from the database, accessing the members and admin will be a breeze. Here's an example:

```python
org_id = "xxx-xxx"
organization = await engine.find_one(OrganizationDB, OrganizationDB.id == ObjectId(org_id))

# Get the admin
print(organization.owner)
# UserDB(...)

# Get the members
print(organization.members)
# [UserDB(...), UserDB(...), ...]
```

Reply:
Alright, would effect that.